eZ/Publish/Core/MVC/Symfony/Matcher/ConfigurableMatcherFactoryInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * File containing the MatcherFactory interface.
```

Comment:
You can skip this, it will be easier in the future to apply common header via cs-fixer.

Reply:
Fixed